website/docs/ocl.md

```diff
@@ -120,25 +120,43 @@ This is the list of security (groups) defined in the service network. Each secur
     "meta": "data",
     "other": true
   },
-  "artifacts": [
-    {
-      "name": "my-artifact",
-      "type": "jar",
-      "url": "mvn:groupId/artifact/1.6",
-      "properties": {
-        "additional": "property",
-        "another": "one"
+  "image": {
+    "provisioners": [{
+      "name": "my-kafka-release",
+      "type": "shell",
```

Comment:
Does it mean that we can have different kind of provisioning (like `ansible`, `shell`, ...) ? It looks good to me.

Reply:
Just a note (not necessary in this PR): it would be great to document the expected behavior/lifecycle in the orchestrator.

Reply:
> Does it mean that we can have different kind of provisioning (like `ansible`, `shell`, ...) ? It looks good to me.

That need a futher discussion. It is possible, but for now,  it would be better to compatible with `packer`,  which we will buiding on.

Reply:
> Just a note (not necessary in this PR): it would be great to document the expected behavior/lifecycle in the orchestrator.

Good idea.

Reply:
:+1 

OK to start with `packer`, but we should do that in kind of abstract way. For instance, you could also `tekton` in some use cases.
Remember the purpose is really to integrate with CSP fundamentals APIs, we have to "limit" the number of prerequisites services for the long term.

Reply:
That's right, the `provisioners.type` here must be abstract. we just keep it similar to `packer`. The translation should be limited in the CSP's plugin.

Reply:
:+1: 

Yes, fully agree ! The CSP plugin will deal with the provisioners ;)